README.md

```diff
@@ -14,6 +15,59 @@ Content for each chapter is split in two subfolders:
 Lecture content is expected to be presented and followed.
 Practice content is expected to be used hands-on individually or as part of team.
 
+## Publishing workflow
+
+Before building the repository you have to set up the following items:
+
+* select the GitHub Pages branch by following the [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) tutorial; you will have to set up the `gh-pages` branch as the publishing branch;
+* add a publishing key named `ACCESS_TOKEN` by following the [Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) tutorial.
+
+In order to publish the content of this repository, we use a GitHub workflow located in the `.github/workflows/deployment.yml`.
+This workflow will build the site using [Docusaurus](https://docusaurus.io/) and publish the contents to a link similar to `<github_username>.github.io`.
+
+## Your first TODOs
+
+To configure this repo for the class for which you intend to develop content, you must make the following changes:
+
+* change the repository name to match your class name, the default is `oer-template`
+* in `config.yaml`:
+  * change `title` to you class name;
+  * change `url` to `<github_usename>.github.io`;
+  * change `baseurl` to `/` if the repository is the only one with a GitHub page in your profile, otherwise, change it to `<repository_name>`;
+  * change links to social media sites, if applicable.
```

Comment:
```suggestion
  * Change `title` to you class name.
  * Change `url` to `<github_usename>.github.io`.
  * Change `baseurl` to `/` if the repository is the only one with a GitHub page in your profile.
     Otherwise, change it to `<repository_name>`.
  * Change links to social media sites, if applicable.
```